main.cpp

```diff
@@ -360,7 +285,35 @@ Thing 3) Smartphone
     2) call Charles
     3) open maps
  */
-
+struct smartPhone
+{
+  float typeOfCpu;
+  float sizeOfStorage;
+  int numbOfSim;
+  float Ram;
+  float screen;
+
+  struct simCard
+  {
+    simCard();
+
+    std::string operatorBrand = "Vodafone";
+    double height = 3.37;
+    double width = 2.12;
+    double depth = 0.029;
+    int productYear = 2015;
+
+    void getAndBuy(double pricing = 77.1);
+    bool unpack = true;
+    void insertSimSlot(int simCard);
+  };
+
+  void insertPhone(bool insertSimSlot = true, bool openPhone = true);
+  void callCharles(bool openCallScreen, int phoneNumber, bool clickCallButton);
+  void openMaps(std::string mapName = "Yandex Maps", bool enter = true);  
+
+  simCard connected;
+};
```

Comment:
3 things need revising here:
- You have 6 variable declarations and 2 function declarations in your nested class.  The instructions specified that 5 variables and 3 functions need to be declared.
- The code inside the `{ }` needs to be indented 4 spaces relative to the `{`
- Type names need to begin with a capitalized first letter. `SmartPhone` and `SimCard` is what you want to write. 

Please correct the issues
- [ ] revise